frontend/src/app/components/containers/complaints/complaint-list-tabs.tsx

```diff
@@ -44,7 +44,7 @@ export const ComplaintListTabs: FC<props> = ({ complaintType, viewType, complain
 
       switch (item) {
         case COMPLAINT_TYPES.ERS:
-          if (UserService.hasRole(Roles.CEEB_COMPLIANCE_COORDINATOR)) {
+          if (UserService.hasRole(Roles.CEEB)) {
```

Comment:
You'll want to include a second check for the coordinator rule: 

```
if(UserService.hasRole(Roles.CEEB) && UserService.hasRole(Roles.CEEB_COMPLIANCE_COORDINATOR))
{
}
```

this will make sure that its a CEEP user with the Coordinator role

Reply:
Acceptance criteria of this ticket does not say that the user has to be also a coordinator.

Reply:
I believe it's good as is.  So long as the user has the CEEB role, it covers the case of either a user with just the CEEB role or a user with both the CEEB and Coordiantor role